src/components/super-cursor.js

```diff
@@ -103,6 +103,18 @@ AFRAME.registerComponent("super-cursor", {
     }
   },
 
+  _isInteractable: function(el) {
+    if (el.className === "interactable") {
+      return true;
+    }
+
+    if (el.parentNode && el.parentNode != el.sceneEl) {
```

Comment:
Hmm, walking up the tree like this every frame is a bit concerning. Also I would inline this recursion. Shouldn't the raycaster be handling this filtering for you anyway?